### ISSUE TITLE
'.vue' file extension does not automatically resolve for component imports.

### ISSUE THREAD

**Issue by VictorTolbert** [nuxt/nuxt.js#59]
For example,

```js
import Widget from '~components/widget'
```

Works for...
```
./components/widget.js
./components/widget.json
./components/widget/index.js
./components/widget/index.json
```

But, not for...
```
./components/widget.vue 
./components/widget/index.vue
```

**Comment by Atinux** [comment]
Hi @VictorTolbert 

It will be fixed in the next release.

Thank you for spotting it 👍

**Comment by Atinux** [comment]
Fixed in [`0.9.0`](https://github.com/nuxt/nuxt.js/releases/tag/v0.9.0) 🔥
